ql/src/java/org/apache/hadoop/hive/ql/parse/ReplicationSpec.java

```diff
@@ -426,4 +427,14 @@ public static void copyLastReplId(Map<String, String> srcParameter, Map<String,
       destParameter.put(ReplicationSpec.KEY.CURR_STATE_ID.toString(), lastReplId);
     }
   }
+
+  public boolean needDupCopyCheck() {
+    return needDupCopyCheck;
+  }
+
+  public void setNeedDupCopyCheck(boolean isFirstIncPending) {
```

Comment:
We can abstract the caller logic that duplicate copy check needed if first incremental pending.
Shall change the name of isFirstIncPending to isNeedDupCopyCheck.
Also, the comment can be moved to caller.

Reply:
i think for now duplicate check is done only for first incremental load ..so the naming looks fine

Reply:
Yes agree. But, from ReplicationSpec point of view we can abstract this logic, so that in future if someone wants to run duplicate check for one more case, this name still remain valid.

Reply:
its abstracted ..now its based on one flag ..alter new flags can be added if logic chnages

Reply:
OK